Configuration_adv.hpp

```diff
@@ -534,6 +534,18 @@
     #endif
 #endif
 
+#if DEW_HEATER == 1
```

Comment:
Let's use the usual `USE_XXXX` format, so rename this to `USE_DEW_HEATER`. Also, I think it might make sense to split heaters, or do we anticipate everyone having both heaters?

Reply:
As there might be users who use DEW_HEATER already, I don't want to do that. Also, it's in a ton of places.

And no, it doesn't make sense. If you don't want two heaters, just don't set the value to the second one.

Reply:
I think there's at best a handful, so we're going to change that regardless and just tell people, maybe we'll have a warning in the code that checks for the existence of that variable. And there is global find and replace so it doesn't really matter how many times it used, lol. Anyway, if you don't want to do it as part of this PR, that's fine, I can change it later for the next release.